utils/helpers.js

```diff
@@ -12,8 +12,19 @@ export const valueOrDefault = (value, defaultValue) =>
 export const formatNumber = (number, { locale = 'en-GB', ...options } = {}) =>
   new Intl.NumberFormat(locale, options).format(number)
 
+export const formatDate = (date, options = {}) => {
+  try {
+    return new Intl.DateTimeFormat('en-GB', options).format(new Date(date))
```

Comment:
Date-s should be converted on the Store side in the future. At the moment it's fine